include/NovelRT.Interop/Input/NrtBasicInteractionRect.h

```diff
@@ -0,0 +1,33 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_INTEROP_INPUT_BASICINTERACTIONRECT_H
+#define NOVELRT_INTEROP_INPUT_BASICINTERACTIONRECT_H
+
+#include "../NrtTransform.h"
+#include "../NrtInteropUtils.h"
+#include "NrtKeyCode.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct BasicInteractionRectHandle* NrtBasicInteractionRect;
+
+NrtResult Nrt_Input_BasicInteractionRect_executeObjectBehaviour(NrtBasicInteractionRect object);
+NrtResult Nrt_Input_BasicInteractionRect_getTransform(NrtBasicInteractionRect object, NrtTransform* outputTransform);
+NrtResult Nrt_Input_BasicInteractionRect_setTransform(NrtBasicInteractionRect object, NrtTransform transform);
+int Nrt_Input_BasicInteractionRect_getLayer(NrtBasicInteractionRect object);
```

Comment:
use proper size here. If one is not provided by the C++ side fix it there too please.